Yosemite/Yosemite/Stores/CardPresentPaymentStore.swift

```diff
@@ -131,7 +135,8 @@ private extension CardPresentPaymentStore {
                         onCompletion(.failure(CardReaderServiceError.paymentCapture()))
                         return
                     }
-                    onCompletion(.success(()))
+                    // This is always failing at this point. 
+                    //onCompletion(.success(receiptParameters))
```

Comment:
Awesome, thanks. I had to comment this out in #4015, otherwise the whole process is reported as an error (which is what we want, only, we don't want it now)